### DISCUSSION TITLE
Vue extension totally not working in chrome with pinia states, how to tell the state?

### DISCUSSION THREAD

**Discussion by dgloriaweb** [vuejs/pinia#2772] [Help and Questions] [answered]
Hi, it's been a while since the chrome vue extension started to have issues showing pinia (at all) and showing the states correctly. I've been expecting it to be fixed for years now, and still not working. Apart from consol logging the states, what ways are there to be sure that the states change as they should? It is crucial, since for  a multi page app storing the states isn't an easy task, and I want to make sure my users' logged in and other status is maintained. Thanks.

**Comment by posva** [comment, marked as answer]
My guess here is that you are using an outdated version of the devtools. Make sure you are using the [latest](https://chromewebstore.google.com/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd). You can also give a try to [the next version](https://devtools.vuejs.org/getting-started/installation)

**Reply by dgloriaweb** [reply]
I've reinstalled it and have the same issue.

**Reply by posva** [reply]
Then it’s probably something else in your extensions or app. Either way, try out the vite plugin, it should work out

**Reply by dgloriaweb** [reply]
vote plugin? If you meant vite, there is no such plugin

**Reply by posva** [reply]
It’s right there in the link I shared

**Reply by dgloriaweb** [reply]
> It’s right there in the link I shared

sorry, I've missed that. Thought it's vue.

**Comment by dgloriaweb** [comment]
Hi, I've tried the vite extension, and that works fine so far.
